api/src/main/scala/com/gu/footballtimemachine/ApiLambda.scala

```diff
@@ -127,12 +139,12 @@ object ApiLambda {
   def main(args: Array[String]): Unit = {
     val req = new ApiGatewayRequest()
     req.setHttpMethod("GET")
-    req.setPath("/competitions/matchDay/apiKey/20170809")
+    req.setPath("/competitions/matchDay/apiKey/20241204")
     req.setQueryStringParameters(Map("startDate" -> "2017-06-11T21:00:00Z", "speed" -> "3").asJava)
```

Comment:
Does the string of the map value need to be updated as well to align with `20241204`?

Reply:
They're used for different methods so it's fine. Though we could make the local testing a bit clearer in a separate PR